typescript/infra/src/xerc20/utils.ts

```diff
@@ -617,6 +730,104 @@ export async function deriveBridgesConfig(
   return bridgesConfig;
 }
 
+export async function deriveWLBridgesConfig(
+  warpDeployConfig: WarpRouteDeployConfig,
+  warpCoreConfig: WarpCoreConfig,
+  multiProvider: MultiProvider,
+): Promise<BridgeConfigWL[]> {
+  const bridgesConfig: BridgeConfigWL[] = [];
```

Comment:
i wonder if there is a way to abstract the limit setting details in the config and have the code do the proper changes based on the underlying xERC20 impl (just rambling you can ignore this comment, but it might interesting to think about it)

Reply:
will punt because refactoring is a large task that will delay updating the limits for this specific warp route